articles/IsolatedFunctions/Readme.md

```
@@ -1,22 +1,345 @@
+Then in the code, configure connectivity to Azure resources conditionally based on the value of this variable.
+
+### Why not use ServiceBusAdministrationClient to Provision Service Bus Resources?
+
+We use TestContainers.ServiceBus to create the Azure Service Bus emulator container.
+
+By design, the emulator does not support management operations, according to the [official documentation](https://learn.microsoft.com/en-us/azure/service-bus-messaging/overview-emulator#known-limitations).
+
+* In the Azure Service Bus Emulator, it only supports data plane operations such as sending and receiving messages
+* It does not support management operations such as creating queues, topics, or subscriptions
+* So to create the service bus resources, you need to use a configuration file,
+  * https://learn.microsoft.com/en-us/azure/service-bus-messaging/test-locally-with-service-bus-emulator?tabs=automated-script#interact-with-the-emulator 
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)